### PR TITLE
operator ibm-security-verify-access-operator (22.10.0)

### PR DESCRIPTION
**Pull request by ashashiv**
Signed-off-by: asha <ashashiv@au1.ibm.com>

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by mvalarh** [comment]
@ashashiv please fix full path image `ibmcom/verify-access-operator:22.10.0` . Probably add registry hostname. Thanks

File: operators/ibm-security-verify-access-operator/22.10.0/manifests/ibm-security-verify-access-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    url: https://www.ibm.com/docs/en/sva
+  maintainers:
+  - email: isamdev@au1.ibm.com
+    name: Verify Access Development Team
+  maturity: stable
+  minKubeVersion: 1.17.0
+  provider:
+    name: IBM
+    url: https://www.ibm.com
+  version: 22.10.0
```
Comment (mvalarh) [inline operators/ibm-security-verify-access-operator/22.10.0/manifests/ibm-security-verify-access-operator.clusterserviceversion.yaml:294]: @ashashiv you are missing replaces `spec.replaces : ibm-security-verify-access-operator.v21.10.0`

**Review by mvalarh — APPROVED** [review]

**Comment by mvalarh** [comment]
@ashashiv waiting for reviewers

**Comment by ashashiv** [comment]
> @ashashiv waiting for reviewers to approve

@mvalarh  Sure, thanks

**Comment by J0zi** [comment]
@isamdev1,@scottexton , please approve as you are original reviewer(s).

**Comment by scottexton** [comment]
@J0zi I approve - I just don't have permission to create and mark a code review request as approved on this PR.

**Comment by mvalarh** [comment]
@scottexton after approve, you can make issue as draft and then `ready for review`. In this way you will get `authorized-changes` label and it will be automerged

**Comment by mvalarh** [comment]
@scottexton i think anybody can do review by going to `Files changed` tab and on top right there is green button `Review changes`, then mark it as approved. After approval one (at least author) can do [following](https://k8s-operatorhub.github.io/community-operators/self-merge-updates/#how-can-i-approve-a-pr-against-my-operator)

**Review by scottexton — COMMENTED** [review]
Reviewed.

**Comment by scottexton** [comment]
@mvalarh Unfortunately I don't have permission to do this.  I don't have an 'approved' option in the code review and can only adda comment.

**Comment by scottexton** [comment]
@mvalarh Here is what I mean:
image

**Pull request merged**